### ISSUE TITLE
Support GET requests on metadata

### ISSUE THREAD

**Issue by wolf31o2** [blinklabs-io/kupogo#6]
Retrieve metadata from `/metadata/{slot-no}`

**Comment by verbotenj** [comment]
Is there any structure expected for metadata?

**Comment by wolf31o2** [comment]
For now, no. There's a huge number of possible structures, but we're trying to capture those in the `cardano-models` repository. For now, just returning the JSON is fine. We can't match it. From Kupo, we'll always get something like:

```go
type Metadata struct {
  hash   string `json:"hash"`
  raw    string `json:"raw"`
  schema map[string]interface{} `json:"schema"`
}
```
The schema interface can be a map, int, string, or bytes.

**Comment by verbotenj** [comment]
Closes with #7
